references/cli.md

```diff
@@ -49,6 +49,9 @@ Usage: substra login [OPTIONS]
   Login to the Substra platform.
 
 Options:
+  --timeout INTEGER               Set timeout (seconds) for retrying a call
+                                  [default: pretty]
```

Comment:
What does pretty, for the default value, mean here ?

Reply:
it's a typo, should be a float